libbeat/processors/add_cloud_metadata/provider_azure_vm_test.go

```diff
@@ -201,7 +201,8 @@ func TestRetrieveAzureMetadata(t *testing.T) {
 			"service": mapstr.M{
 				"name": "Virtual Machines",
 			},
-			"region": "eastus",
```

Comment:
CAn you try 
logger := logp.NewTestingLogger() in line 166?
Maybe this will fix the lint issue